pallets/demeter-farming-platform/src/lib.rs

```diff
@@ -785,9 +785,6 @@ pub mod pallet {
                 return Err(Error::<T>::Unauthorized.into());
             }
 
-            // Check if multiplier is valid
-            ensure!(new_multiplier >= 1, Error::<T>::InvalidMultiplier);
```

Comment:
Can new_multiplier be 0?

Reply:
Yes, that is why I created this PR. We need that change.